### PR TITLE
Update Conviva Android dependency

### PR DESCRIPTION
**Pull request by testcenter**
## Changes
Update to the latest Android dependency

## Checklist
- [x] 🗒 `CHANGELOG` entry

### CODE REVIEW

**Review by rolandkakonyi — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitmovin-player-react-native-analytics-conviva",
-  "version": "1.3.0",
+  "version": "1.4.0",
```
Comment (rolandkakonyi) [inline package.json:3]: please revert this
```suggestion
  "version": "1.3.0",
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [1.4.0] - 2024-09-05
```
Comment (rolandkakonyi) [inline CHANGELOG.md:3]: Please use `[Unreleased]`, the release automation will update it

```suggestion
## [Unreleased]
```

**Review by rolandkakonyi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (testcenter) [reply CHANGELOG.md]: [912f10f](https://github.com/bitmovin/bitmovin-player-react-native-analytics-conviva/pull/29/commits/912f10f218efd80288b98bb9b0a6f210b670b19d)

File: package.json (hunk 1 above)
Comment (testcenter) [reply package.json]: [912f10f](https://github.com/bitmovin/bitmovin-player-react-native-analytics-conviva/pull/29/commits/912f10f218efd80288b98bb9b0a6f210b670b19d)

**Review by rolandkakonyi — APPROVED** [review]

**Review by rolandkakonyi — APPROVED** [review]

**Pull request merged**